mappings/net/minecraft/client/ClientBrandRetriever.mapping

```diff
@@ -1,2 +1,2 @@
 CLASS net/minecraft/client/ClientBrandRetriever net/minecraft/client/ClientBrandRetriever
-	METHOD getClientModName ()Ljava/lang/String;
```

Comment:
Oh god this looks like another one of those changes that will cause the game to crash. TBD